zips/zip-0234.md

```diff
@@ -76,8 +76,8 @@ Key Objectives:
 
 1. We want to introduce an automated mechanism that allows users of the network
    to contribute to the long-term sustainability of the network.
-2. We want to enable ZEC that has been burned to be recreated in the future to
-   benefit network sustainability.
+2. We want to enable ZEC that has been removed from circulation to be recreated
```

Comment:
Minor technical note: ZEC can only be issued (and therefore reissued); the notes it is stored in are what get created and spent.
```suggestion
2. We want to enable ZEC that has been removed from circulation to be reissued
```